wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -245,6 +245,10 @@ executable cardano-integration-test
                        Types
                        Functions
                        Error
+                       Util
+                       WalletSpecs
+                       AddressSpecs
+                       TransactionSpecs
```

Comment:
Why split it into four modules? I feel like a single module would be fine.

Reply:
I respectfully disagree 😉 Now it might look like it doesn't make sense to do so, but if in the future (and that's the goal I have in mind) we will have several dozens of tests, I feel that modifying just a monolith file is not the way to go, imho. 

Even just for the sake of contributing to the specs, different developers can contribute to different specs without merge conflicts every time they open a PR. The "glance" effect is rapidly lost once you start staking 20,30,50 specs in a file.

Said that, I would stick with democratic consensus, so I would pull @erikd as he has a great taste in these kind of things and he knows about testing more than I do anyway 😀

Reply:
I don't have any strong opinions either way, but in general I don't like doing `X` now because we might do `Y` later. I usually prefer to do `X` when I *know* that the next thing I am going to do is `Y`.

Reply:
> I usually prefer to do X when I know that the next thing I am going to do is Y.

That is what I was going to do next anyway (write more specs), but on the other hand I don't want to rub Matt the wrong way by trying to advocate my solution over his. Let's see what me and Matt can decide when he gets online and we'll take from there 😀 

Cheers Erik!

Reply:
Side note but:

*.git/hooks/post-checkout*
```bash
#!/bin/sh
.git/hooks/ctags >/dev/null 2>&1 &
```

and 

*.git/hooks/ctags*
```bash
#!/bin/sh
set -e
hasktags --ctags --ignore-close-implementation .
```

to generate and refresh ctags does wonderful things for me with vim and everything going with the `:tags` feature. Not sure what editor you use it helps quite a lot to jump around files in the codebase.

Reply:
I use and recommend `codex` so you get tags for dependencies, too :) but having a git hook for this is a great idea!

Reply:
> We have a 1746 line monolithic API.V1.Types file 

Wow, when was the last time you people coded in C or C++?? 1746 lines is not that big :)

Reply:
... never? :sweat_smile: haskell 4 lyf